tests/test_core/test_napp_dir_listener.py

```diff
@@ -0,0 +1,26 @@
+"""Test kytos.core.napps.napps_dir_listener module."""
+from unittest import TestCase
+from unittest.mock import Mock
+
+from kytos.core.napps.napps_dir_listener import NAppDirListener
+
+
+class TestNAppDirListener(TestCase):
+    """NAppDirListener tests."""
+
+    def setUp(self):
+        """Method executed before each test."""
+        self.controller = Mock()
+        self.controller.options.napps = '/tmp'
+        self.napp_dir_listener = NAppDirListener(self.controller)
+        self.event = Mock(src_path='/tmp/username/napp_name/')
+
+    def test_on_created(self):
+        """Test whether on_created is calling load_napp."""
+        self.napp_dir_listener.on_created(self.event)
```

Comment:
@cemsbr is this test ok? Shouldn't we test this by creating a directory under the 'napps_dir' (mocked) to see the event being dispatched by the observer?